### ISSUE TITLE
some questions about extracting I3D features

### ISSUE THREAD

**Issue by liucun12** [wjtan99/BERT_Anomaly_Video_Classification#1]
Will you extract I3D features of 10crop? If you have code, can you provide me with a copy? Thank you very much!

**Comment by wjtan99** [comment]
I used https://github.com/piergiaj/pytorch-i3d for feature extraction.  For 10-crop, refer to the following code,  

#04/24 added tencrop 
mean = [0.43216, 0.394666, 0.37645] 
std = [0.22803, 0.22145, 0.216989]  
#05/04/22 for I3D 
mean = [0.5, 0.5, 0.5] 
std = [0.5, 0.5, 0.5]  

normalize = transforms.Normalize(mean=mean, std=std)

test_transforms = transforms.Compose([
    transforms.TenCrop(224), # this is a list of PIL Images
    transforms.Lambda(lambda crops: [transforms.ToTensor()(crop) for crop in crops]),
    transforms.Lambda(lambda crops: [normalize(crop) for crop in crops]),
    transforms.Lambda(lambda crops: torch.stack(crops))
])
After you read in an image, apply:   
    img_out = test_transforms (img)
